dvclive/metrics.py

```diff
@@ -45,9 +45,7 @@ def __init__(
             try:
                 os.makedirs(self.dir, exist_ok=True)
             except Exception as exception:
-                raise DvcLiveError(
-                    "dvc-live cannot create log dir - '{}'".format(self.dir),
-                ) from exception
+                raise CannotCreateLogDirError(self.dir) from exception
```

Comment:
Not sure if we really want to raise a custom exception here as it seems that it would be related to Python specific errors and not with any dvclive specific logic. 

What would be the scenarios where this exception will be raised?? I can think of having a `PermissionError` where I think the default Python exception would be already clear enough.

cc @pared

Reply:
yep, you're right, only `PermissionError` can occure

Reply:
So maybe we could remove `CannotCreateLogDirError` as part of this P.R. (still makes sense under the scope of "get rid of DvcLiveError calls")  and revisit it for considering a custom exception in the future

Reply:
So we don't need to raise `PermissionError`, and write a test for it ?

Reply:
Yep. Just let that be handled by Python